### ISSUE TITLE
No respect for notification settings

### ISSUE THREAD

**Issue by alexander-potemkin** [mattermost/desktop#1712]
I confirm (by marking "x" in the [ ] below: [x]):

- [ X ] This is not a troubleshooting question. [Troubleshooting questions go here: http://www.mattermost.org/troubleshoot/](http://www.mattermost.org/troubleshoot/).
- [ X ] This doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [ X ] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md).

---

**Summary**
I unchecked both "Flash app windows..." and "Show icon in the notification area" but the icon is still flashes and the icon is still there. Tried to set up them up and remove again, but with no luck.

**Environment**
- Operating System: Linux Mint 20.02 (uma)
- Mattermost Desktop App version: 4.7.1
- Mattermost Server version:  5.39.2

**Steps to reproduce**
- setup fresh mattermost from the deb file on latest Linux Mint (Cinnamon)
- uncheck those options

**Expected behavior**
- no icon in tray
- icon doesn't flash when new notifications

**Observed behavior**
- icon tray is on tray
- icon flash on every new notification

**Possible fixes**

It seems like those settings are not respected somehow.
![Screenshot from 2021-08-30 15-27-26](https://user-images.githubusercontent.com/2323840/131338893-73aa0c4b-5490-44a0-b7a1-f22b737e462f.png)

**Comment by alexander-potemkin** [comment]
And here is the .config/MatterMost/config.json

```
{
  "version": 2,
  "teams": [
    {
      "name": "<...>",
      "url": "<...>",
      "order": 0
    }
  ],
  "showTrayIcon": true,
  "trayIconTheme": "dark",
  "minimizeToTray": true,
  "notifications": {
    "flashWindow": 2,
    "bounceIcon": true,
    "bounceIconType": "informational"
  },
  "showUnreadBadge": true,
  "useSpellChecker": true,
  "enableHardwareAcceleration": true,
  "autostart": true,
  "spellCheckerLocale": "en-US",
  "darkMode": false,
  "downloadLocation": "/home/<...>/Downloads"
}
```

Would be happy to change settings manually as a work-around, just - please, advice on the values

**Comment by amyblais** [comment]
Taskbar flashing issue has a ticket here https://mattermost.atlassian.net/browse/MM-37797.
@Willyfrog Let me know if we're able to test the other issue mentioned here.

**Comment by alexander-potemkin** [comment]
@amyblais , @devinbinnie , thank you very much for your prompt responce and apologies for the delay in getting back to you!
Glad to know it's acknowledged and fixed, unfortunately, I can't try it now - hate to ruin my auto-updated setup, is there any time estimate on when 4.7.2 is expected to be released?

**Comment by amyblais** [comment]
Final testing is in progress, so I'd estimate it may be released this week.

**Comment by alexander-potemkin** [comment]
Thank you.
Just one more thing: it's turned out that the process doesn't quit, when I close the window, and no tray icon either. It's also memory leaking like crazy: it starts with 100Mb and ends up with 300Mb+ in just a couple of days, not so high loaded use.

**Comment by Willyfrog** [comment]
Hi @alexander-potemkin can you fill a separate issue for that problem, if you can add any screenshot/details you may have that would be superhelful in trying to replicate the problem.

**Comment by amyblais** [comment]
Original report is fixed in v4.7.2,
